content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,122 @@
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-104/python.png" src="/img/blogs/newsletter-104/python.png" caption="Running Python 3.7 on ReactOS (this is used to run the Winesync python script)">}}
+{{< /gallery >}}
+
+The aim of this effort is to bring the improvements and fixes from WINE that help with the support of NT6+ software that otherwise wouldn't have worked.
+This also helps us remove older hacks that are no longer needed.
+
+## Asynchronous network connection support
+ReactOS supports basic networking provided by the Sockets Provider API (MSAFD), that is, applications that require an Internet connection can perform networking operations in a blocking way.
+This means that the connection is done between peer-to-peer synchronously, which has many limitations.
+Nowadays modern applications perform network operations asynchronously (aka non-blocking way) which ReactOS currently does not support.
+
+This leads to undefined behaviors and bugs with certain network related API function calls such as `WSPConnect` that do not work properly when required to estabilish a network connection of a socket
+to a peer and such in non-blocking (asynchronous) way.
+Thamatip Chitpong (TAN-Gaming) aims to fix this problem by taking the lead on working on an implementation for asynchronous networking support ([PR #6349](https://github.com/reactos/reactos/pull/6349)).
+During his development he also fixed other miscellaneous bugs and issues related to the MSAFD component.
+
+This greatly improves the network reliability of applications such as web browsers, FTP file transfers, multi-player games and whatnot.
+Although the performance is not on par with Windows yet, Thamatip has future plans to tackle this and other problems not currently handled by his PR.
+
+## Audio improvements
+Several major improvements in audio subsystem have been made by Oleg Dubinskiy recently.
+In [PR #6686](https://github.com/reactos/reactos/pull/6686), he implemented handling modern audio formats (e.g. Extensible audio, Float audio, and others). He also implemented support for high sample rates, setting bits per sample sound quality, and more than two (stereo) output channels.
+
+{{< video src="/img/blogs/newsletter-104/ros_extensible_format.mp4" type="video/mp4" caption="Testing the extensible audio format of some apps">}}
+
+This allows many 3rd-party programs and games (e.g. AIMP 5.30, QMMP 0.12.17, all Chromium based browsers, and the Game Dev Tycoon) which are using modern formats, to open an audio device and play sounds correctly.
+However, the sound only works when the audio miniport driver (e. g. Intel AC97, Realtek HD Audio etc.) supports the requested format directly.
+Otherwise it won't work, because playing a format which is not supported by the driver requires resampling support, which is currently not finished; so this feature is disabled in ReactOS for now.
```

Comment:
```suggestion
Otherwise it doesn't work, because playing such format would require resampling support, which is currently not finished; so this feature is disabled in ReactOS for now.
```

Reply:
Won't (will not) is better here than doesn't (does not). Also, it should be "such a format..." not "such format"

Reply:
Alright, but "will not" sounds like "never" for me, which is not the case, because "until resampling implemented".